Gopkg.lock

```diff
@@ -4,13 +4,21 @@
 [[projects]]
   name = "github.com/DataDog/agent-payload"
   packages = ["gogen"]
-  revision = "b0b883b8c68f11698e1eee462d1efa83e76bf14a"
-  version = "4.6"
+  revision = "d6a2f74c00f08246bc0ca160ffa9d28756968065"
+  version = "4.5"
```

Comment:
if there's no plan to re-introduce the extra sketch types it could make sense to remove them from the payload format and bump the version of the format. What do you think?

Reply:
I moved the version back to the one without the extra sketches.